addons/vehicles/XEH_postInit.sqf

```diff
@@ -1,7 +1,42 @@
 // by esteldunedain
 #include "script_component.hpp"
 
+// eject destroyed vehicle init
+[QGVAR(ejectDestroyed),{
+    TRACE_2("ejectDestroyed event",_this,typeOf _this);
+    _this removeEventHandler ["HandleDamage", _this getVariable QGVAR(ejectDestroyedHDEH)];
+    _this setVariable [QGVAR(ejectDestroyedHDEH), nil];
+}] call CBA_fnc_addEventHandler;
+private _ejectDestroyedClasses = [];
+{
+    private _ejectDestroyedClass = configName _x;
+    if (-1 == _ejectDestroyedClasses findIf {_ejectDestroyedClass isKindOf _x}) then {
```

Comment:
How does this not error with array to number comparison?

Reply:
This wasn't meant as a criticism, I'm just a curious boi 😆

My headcanon interpretation of sqf doesn't fit with this line because it would error as soon as it hit the array value after the `==`

Reply:
`findIf` returns index or -1 if not found

Reply:
Yeah I get that, I'm just so used to interpreting SQF left to right. It seems `findIf` has higher operator precedence than `==` 👍

Reply:
@SilentSpike Actually == has lower precedence than the rest, same as all other comparison operators
&&/and are one lower than ==
and ||/or are the lowest.

Reply:
Thanks dedmen, TIL 👨‍🎓